app/components/cards.hbs

```diff
@@ -0,0 +1,50 @@
+<section class='joining'>
+  <h2 class='joining__title'>Why you should consider joining?</h2>
```

Comment:
Can we please move string data in this file into the constant file as well?

Reply:
That would be oversimplication I guess

Reply:
Not really it's good to have. But your call.